app/models/concerns/normalize_column.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module NormalizeColumn
+  extend ActiveSupport::Concern
+
+  included do
+    def self.normalize_column(column_name)
+      before_validation -> do
+        value = self[column_name]
+        next unless value
```

Comment:
```suggestion
        next if value.blank?
```

Should save us from checking the database is value is blank.

Reply:
@vlado As-in when it's an empty string?

Reply:
Yes, someone could submit the form with empty city or just press space(s) in it (`""`, `" "`, `"       "`). It will not be saved to database and user will see validation error message but we will still do the database call.

If we change it to `next if value.blank?` it will exit early instead.

Reply:
And shouldn't it be `return` instead of the `next`?

Reply:
Since it's lambda, it has to be `next`. It's `return` in the last changes.